src/Voron/Impl/FreeSpace/FreeSpaceHandling.cs

```diff
@@ -60,7 +64,12 @@ public static bool IsFreeSpaceTreeName(string name)
 
                     if (num < NumberOfPagesInSection)
                     {
-                        return TryFindSmallValue(tx, freeSpaceTree, it, num);
+                        (int count, int count2) counts = (0, 0);
+                        var stop = Stopwatch.StartNew();
+                        long? pageFound = TryFindSmallValue(tx, freeSpaceTree, it, num, ref counts);
+                        if(Logger.IsInfoEnabled && counts.count > 10)
```

Comment:
Let's skip the logging if `lowerIdSlice` has no value.

Reply:
Why? It shouldn't happen (only in tests) and if it does I prefer to know